Scala/Scala.sublime-syntax

```diff
@@ -440,16 +578,123 @@ contexts:
       push:
         - match: '`'
           pop: true
-    - match: '\b[a-z][a-zA-Z0-9_]*\.'
-    - match: '\.[a-z][a-zA-Z0-9_]*\b'
+    - match: '{{varid}}\.'
+    - match: '\.{{varid}}'
     - match: '\b{{varid}}'
       scope: entity.name.parameter.scala
+    - match: \[
+      push:
+        - match: \]
+          pop: true
+        - include: main
+    - match: '@|_'
```

Comment:
It depends on the context.  In pattern match position, it is a convention for an unused variable except when it is followed by a `*`, in which case it denotes a varargs extraction.  However, because `_` is not a valid *dereference* identifier, it's a slightly stronger convention than one would normally assume.  Also, there is no checking for repeated use of `_` in patterns, whereas a normal identifier would be forbidden.

So in other words, yeah, it really is a keyword, just a particularly odd one.